src/libraries/System.Net.Http/src/System/Net/Http/HttpClientHandler.AnyMobile.cs

```diff
@@ -0,0 +1,429 @@
+            // 'Transfer-Encoding: chunked' request headers were specified. So, the handler thus needed to buffer
+            // in the request content to determine its length and then would choose 'Content-Length' semantics when
+            // POST'ing. In .NET Core, the handler will resolve the ambiguity by always choosing
+            // 'Transfer-Encoding: chunked'. The handler will never automatically buffer in the request content.
+            get
+            {
+                return 0; // Returning zero is appropriate since in .NET Framework it means no limit.
+            }
+
+            set
+            {
+                if (value < 0)
+                {
+                    throw new ArgumentOutOfRangeException(nameof(value));
+                }
+
+                if (value > HttpContent.MaxBufferSize)
+                {
+                    throw new ArgumentOutOfRangeException(nameof(value), value,
+                        SR.Format(CultureInfo.InvariantCulture, SR.net_http_content_buffersize_limit,
+                        HttpContent.MaxBufferSize));
+                }
+
+                CheckDisposed();
+
+                // No-op on property setter.
+            }
+        }
+
+        [UnsupportedOSPlatform("android")]
+        [UnsupportedOSPlatform("browser")]
+        [UnsupportedOSPlatform("ios")]
+        [UnsupportedOSPlatform("tvos")]
+        [UnsupportedOSPlatform("maccatalyst")]
+        public int MaxResponseHeadersLength
+        {
+            get => throw new PlatformNotSupportedException();
+            set => throw new PlatformNotSupportedException();
+        }
+
+        [UnsupportedOSPlatform("android")]
+        [UnsupportedOSPlatform("ios")]
+        [UnsupportedOSPlatform("tvos")]
+        [UnsupportedOSPlatform("maccatalyst")]
+        public ClientCertificateOption ClientCertificateOptions
+        {
+            get => throw new PlatformNotSupportedException();
+            set => throw new PlatformNotSupportedException();
+        }
+
+        [UnsupportedOSPlatform("android")]
+        [UnsupportedOSPlatform("browser")]
+        [UnsupportedOSPlatform("ios")]
+        [UnsupportedOSPlatform("tvos")]
+        [UnsupportedOSPlatform("maccatalyst")]
+        public X509CertificateCollection ClientCertificates
+        {
+            get
+            {
+                throw new PlatformNotSupportedException();
+            }
+        }
+
+        [UnsupportedOSPlatform("android")]
+        [UnsupportedOSPlatform("browser")]
+        [UnsupportedOSPlatform("ios")]
+        [UnsupportedOSPlatform("tvos")]
+        [UnsupportedOSPlatform("maccatalyst")]
+        public Func<HttpRequestMessage, X509Certificate2?, X509Chain?, SslPolicyErrors, bool>? ServerCertificateCustomValidationCallback
+        {
+            get => throw new PlatformNotSupportedException();
+            set => throw new PlatformNotSupportedException();
+        }
```

Comment:
This would be nice to have on Apple platforms. There's an underlying API with slightly different signature that is missing the `X509Certificate2` wrapping and passes in the native Apple objects instead. In some earlier Xamarin issue/PR there was a code to do conversion between the two worlds. The reason it was not done directly was because of code size issues.

If this PR gets in then we should open a tracking issue to map this property (and possibly others too). Just putting it here as a reminder.